ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,61 +13,76 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class="has-top-margin-xl">
-  <p class="is-label">Which secrets would you like us to sync?</p>
-  <p class="sub-text">
-    Select a KV engine mount and path to sync a secret to the destination.
-  </p>
-
+<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
   <MessageError @errorMessage={{this.error}} />
 
+  {{#if this.syncedSecret}}
+    <Hds::Alert @type="inline" @color="success" as |A|>
```

Comment:
It also might be helpful to add a loading icon when the sync task is running?

Reply:
I think that's a good idea if we keep it as undismissable. Or perhaps we hide it while the request is in flight and then show it again?

Reply:
Yeah - I think hiding if the task is running makes sense to me!